credentials-considerations.md

```diff
@@ -123,7 +123,7 @@ This is a specific instance of the [basic principles of data minimization](https
 
 Identity systems which are designed to enable collection of only the minimal information necessary to complete a task are less risky than those which supply additional information. For example, a website which requires a copy of the user's full driver's license for access to content universally considered to be adult-only is riskier than one which requests only a cryptographic signature proving the user is in possession of a driver's license with an age over 18 issued by one of a set of trusted authorities. Even better is if the RP is unable to learn which authority issued the credential, such as by having it signed by a trusted intermediary.
 
-See also: [abstract claims](https://www.w3.org/TR/vc-data-model/#favor-abstract-claims); [predicate proofs](https://www.w3.org/TR/vc-imp-guide/#predicates).
+See also: [abstract claims](https://w3c.github.io/vc-data-model/#favor-abstract-claims); [predicate proofs](https://w3c.github.io/vc-imp-guide/#predicates).
```

Comment:
Were you updating this just because the linked version here is a bit behind at the moment? I think we want to keep this pointed at the TR namespace still. When I was editing the VC spec the GitHub link was managed as a kind of staging ground and can sometimes end up pointing out of date.

I believe right now TR is a bit behind just because they're waiting for the publishing of it still, but still tends to be the better link to reference for longevity sake.

Reply:
Yeah as a living document I assumed we'd want to point to the living version of other docs, but I don't have a strong opinion. Happy to leave at the TR version if others think that's better.

Reply:
I believe this is a note and I can't remember what the rules are for a note. I can see the benefit to keeping this pointed at the living version since the space is still fairly active. I think if this weren't a note we'd have more restrictions on how it's meant to be.

In any case, I'm not overly opinionated here either, so consider this more of a nit.